src/handlers/psp.ts

```diff
@@ -213,121 +236,80 @@ const executeApiCalls = async (state: State): Promise<State> => {
   return { ...state, apiValuesBySubscriptionId };
 };
 
-const initializeProviders = async (state: State): Promise<State> => {
-  const { config, baseLogOptions } = state;
-
-  const airnodeWallet = ethers.Wallet.fromMnemonic(config.nodeSettings.airnodeWalletMnemonic);
-
-  const evmChains = config.chains.filter((chain) => chain.type === 'evm');
-  if (isEmpty(evmChains)) {
-    throw new Error('One or more evm compatible chains must be defined in the provided config');
+export const spawn = ({
```

Comment:
I guess it would be nice to move this function into it's own file similar to aws.ts in node package

Reply:
Good point, how about a `workers` folder like in Airnode?